src/com/facebook/buck/cxx/FrameworkPathArg.java

```diff
@@ -23,8 +23,8 @@
 import java.util.Objects;
 
 /** A base class for {@link Arg}s which wrap a {@link FrameworkPath}. */
-abstract class FrameworkPathArg implements Arg {
-  @AddToRuleKey final ImmutableSortedSet<FrameworkPath> frameworkPaths;
+public abstract class FrameworkPathArg implements Arg {
```

Comment:
Made it `public` so that `com/facebook/buck/swift/SwiftLibraryDescription` can create a subclass of it.